cpu/lpc2387/include/periph_cpu.h

```diff
@@ -19,15 +19,13 @@
 #ifndef PERIPH_CPU_H
 #define PERIPH_CPU_H
 
-#include "cpu.h"
-#include "periph/dev_enums.h"
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 #include <stdint.h>
 #include "cpu.h"
+#include "kernel_defines.h"
```

Comment:
Why is this needed?

Reply:
That was for `ARRAY_SIZE()` but since that didn't work with preprocessor arithmetic comparisons, I had to remove it again.